### ISSUE TITLE
Edge lists silently not exported with Ogre > 1.9

### ISSUE THREAD

**Issue by shedatc** [OGRECave/blender2ogre#135]
I'm trying to render a basic scene using Ogre 13.3.1 but shadows are missing:

![No call to buildEdgeList](https://user-images.githubusercontent.com/11380074/159790683-dc103fd7-9ffd-4693-b08f-a8b50725db3e.png)

The scene is built using Blender 3.0.1 and exported with the *Edge lists* box checked:

![export-mesh-setup](https://user-images.githubusercontent.com/11380074/159791797-19bb5f97-3dbc-4262-9f4f-f4035a7b77ed.png)

Ogre's shadow technique is set to `SHADOWTYPE_STENCIL_ADDITIVE` (see `setupShadows` in `sandbox/src/Bootstrap.cpp`).

Explicitly calling buildEdgeList (see `setupScene` in `sandbox/src/Bootstrap.cpp`) trigger shadow rendering:

![With call to buildEdgeList](https://user-images.githubusercontent.com/11380074/159790689-e6b68a1d-5dbe-48c5-b1ef-4b86503cd3ac.png)

What did I miss?

*Source code and material: [sandbox.tar.gz](https://github.com/OGRECave/blender2ogre/files/8336572/sandbox.tar.gz)*

**Comment by paroj** [comment]
this is currently not implemented for Ogre > 1.9. This functionality was removed from OgreXMLConverter and we would need to call OgreMeshUpgrader separately to generate the edge-lists.

**Comment by shedatc** [comment]
IIUC, the _Edge Lists_ checkbox silently do nothing when OgreXMLConverter is taken from Ogre > 1.9, right? And manually using `OgreMeshUpgrader` will add the edge lists to the mesh, preventing me from having call `buildEdgeList`, still right?

**Comment by shedatc** [comment]
> And manually using `OgreMeshUpgrader` will add the edge lists to the mesh, preventing me from having call `buildEdgeList`, still right?

Replying to myself: manually calling `OgreMeshUpgrader` on the `.mesh` file properly generate the edge lists:
```
…
Generating edge lists...
Generating edge lists... success
MeshSerializer writing mesh data to stream Grid.001.mesh...
…
Exporting edge lists...
Edge lists exported
…
```

**Comment by shedatc** [comment]
Also note that the _Use OgreMesh Tools_ checkbox (under the _LOD_ panel) was checked. So I'll investigate the possibility `blender2ogre` just didn't call `OgreMeshUpgrader`.

**Comment by sercero** [comment]
@shedatc `blender2ogre` calls `OgreMeshUpgrader` when that box is ticked, but with the unintuitive -e option (DON'T generate edge lists).

https://github.com/OGRECave/blender2ogre/blob/6f73e4efd03f709c59c3114ef0eab42919c8d743/io_ogre/util.py#L111-L112

It seems that at some point `blender2ogre` created the Edge Lists directly from `mesh.py`, but I can't find that kind of code anywhere in `mesh.py`

**Comment by sercero** [comment]
This is solved by #162
